db/migrate/20250103145109_add_beneficiary_phone_number_to_callout_participations.rb

```diff
@@ -0,0 +1,26 @@
+class AddBeneficiaryPhoneNumberToCalloutParticipations < ActiveRecord::Migration[8.0]
+  def up
+    add_column :callout_participations, :beneficiary_phone_number, :string
+    execute <<-SQL
```

Comment:
Maybe move this to a separate rake task for migrating the data. There are probably millions of callout participations so this might take a long time and the app will be down while this migration is running